barretenberg/cpp/src/barretenberg/honk/composer/goblin_ultra_composer.test.cpp

```diff
@@ -92,18 +99,27 @@ TEST_F(GoblinUltraHonkComposerTests, SingleCircuit)
     // Add mock data to op queue to simulate interaction with a previous circuit
     op_queue->populate_with_mock_initital_data();
 
-    // Construct a test circuit then generate and verify its proof
-    auto verified = construct_test_circuit_then_generate_and_verify_proof(op_queue);
+    auto builder = proof_system::GoblinUltraCircuitBuilder(op_queue);
+
+    generate_test_circuit(builder);
 
-    EXPECT_EQ(verified, true);
+    auto composer = GoblinUltraComposer();
+
+    // Construct and verify Honk proof
+    auto honk_verified = construct_and_verify_honk_proof(composer, builder);
+    EXPECT_TRUE(honk_verified);
+
+    // Construct and verify Goblin ECC op queue Merge its proof
```

Comment:
Did you want to write "Merge its proof"?

Reply:
Corrected, thanks